common/data/sentencePlanningApi.test.js

```diff
@@ -43,20 +43,20 @@ describe('sentencePlanningApi', () => {
     nock.cleanAll()
   })
   let mockedEndpoint
-  const token = '1234'
+  const tokens = { authorisationToken: 'mytoken', sessionKey: 'mysessionkey' }
```

Comment:
I think there is a case to be made for having some test constants. That `mockPromise` function is another good candidate. Shall I raise a tech debt ticket?

Reply:
Yes plz - I agree

Reply:
SP-590 F/E Tech debt - Create test constants raised